taipy/rest/api/resources/scenario.py

```diff
@@ -53,7 +53,7 @@ class ScenarioResource(Resource):
                 In this example the REST API is served on port 5000 on localhost. We are using curl command line
                 client.
 
-                `SCENARIO_63cb358d-5834-4d73-84e4-a6343df5e08c` is the value of the *scenario_id* parameter. It
+                `SCENARIO_63cb358d-5834-4d73-84e4-a6343df5e08c` is the value of the *scenario_id* argument. It
```

Comment:
Please revert.

Reply:
yep working on these changes, will update the PR once done.

Reply:
Thank you very much.